lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -298,6 +309,68 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  /**
+   * Render the control to filter the audits by metric. The filtering is done at runtime by CSS only
+   * @param {LH.ReportResult.AuditRef[]} filterableMetrics
+   * @param {HTMLDivElement} element
+   */
+  renderMetricAuditFilter(filterableMetrics, element) {
+    filterableMetrics.unshift(/** @type {LH.ReportResult.AuditRef} */ ({acronym: 'All'}));
+    // thx https://codepen.io/surjithctly/pen/weEJvX
+    const filterBarEl = this.dom.createElement('div', 'lh-filterbar');
+    const filterBarSummaryEl = this.dom.createChildOf(filterBarEl, 'span', 'lh-filterbar__summary');
+    filterBarSummaryEl.textContent = 'Filter to relevant audits: ';
+    const labelSelectors = [];
+    const auditSelectors = [];
+    for (const metric of filterableMetrics) {
```

Comment:
```suggestion
    for (const metric of [{acronym: 'All'}, ...filterableMetrics]) {
```

(don't mutate input parameters)

alt: `filterableMetric = [{blah}, ...filterableMetric]` up top